wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/WRadioButtonSubmitOnChangeExample.java

```diff
@@ -70,8 +70,6 @@ public void execute(final ActionEvent event) {
 		regionPanel.add(new WHeading(WHeading.SECTION, "Region"));
 		regionPanel.add(regionSelector);
 
-		regionSelector.setAccessibleText("Region");

```

Comment:
Instead of deleting this line could you set a reasonable tool tip. Perhaps something like `regionSelector.setToolTip("Select a region");`